examples/trip_planner/use_planner.py

```diff
@@ -15,6 +15,9 @@
 
 # Please use your own file path
 OUTPUT_FILE_PATH = "trip.md"
+AGENT_SELECTION_CITY_ROLE = """An expert in analyzing travel data to pick ideal destinations. Goal is to help select the best city for a trip based on specific criteria such as weather patterns, seasonal events, and travel costs."""  # noqa: E501
```

Comment:
This is a simple single line string, so please use " instead of """ with proper formatting and remove `# noqa: E501`. Please do all such cases